### ISSUE TITLE
Update the `kytos.json` attributes

### ISSUE THREAD

**Issue by diraol** [kytos/napps-server#76]
```json
{
  "author": "{{author}}",
  "name": "{{napp}}",
  "description": "{{description}}",
  "long_description": "",
  "version": "",
  "napp_dependencies": [],
  "license": "",
  "tags": [],
  "url": "{{url}}"
}
```

**Comment by diraol** [comment]
@beraldoleal, which are the required attributes (to be checked while uploading a NApp)?

**Comment by diraol** [comment]
Fixed at c2df5a1a19d08be6b5bdd6a25f22f9a7c05fccf3
